src/brasil/gov/tiles/tiles/templates/videogallery.pt

```diff
@@ -4,96 +4,99 @@
-                  <p class="description" tal:content="item/description"></p>
+          <div class="galery-items"
+               tal:define="last_item python:items[-1] if items else None">
+            <div class="galery-items-wrapper">
+              <tal:items repeat="item items">
+                <div class="gallery-element" tal:attributes="class python:'gallery-element' + (' selected' if repeat['item'].start else '') + (' last' if item == last_item else '')">
+                  <div class="gallery-element-wrapper">
+                    <a tal:attributes="href item/absolute_url;
+                                       data-player-dom item/embed_html;
+                                       id python: 'gallerylink-' + str(view.data['uuid_container']) + '-' + str(view.get_uuid(item));"
+                       class="gallery-element-link">
+                        <tal:image define="thumbnail python:view.thumbnail(item);"
+                                   condition="thumbnail">
+                          <img tal:define="uuid python:view.get_uuid(item);"
+                               tal:condition="python:thumbnail and uuid"
+                               tal:attributes="src thumbnail/url;
+                                               width thumbnail/width;
+                                               height thumbnail/height;
+                                               data-title string:<a href='${item/absolute_url}/view'>${item/title}</a>;
+                                               data-description item/Description;
+                                               data-link string:${item/absolute_url}/view;
+                                               data-uuid uuid;
+                                               alt python: thumbnail.context.Title()" />
+                        </tal:image>
+                    </a>
+                    <div class="gallery-element-metadata">
+                      <span class="title" tal:content="item/title"></span>
+                      <p class="description" tal:content="item/description"></p>
+                    </div>
+                  </div>
                 </div>
-              </div>
+              </tal:items>
             </div>
-          </tal:items>
+          <div class="visualClear"></div>
+        </div>
+        <div class="visualClear"></div>
+
         </div>
-      <div class="visualClear"></div>
-    </div>
-    <div class="visualClear"></div>
 
-    </div>
-    <div class="gallery-footer">
-      <a href="" tal:content="view/data/footer_text"
-                tal:attributes="href collection/absolute_url"></a></div>
-  </div>
-  <noscript>
-      <div class="error" i18n:domain="brasil.gov.tiles" i18n:translate="">Your internet browser this not support JavaScript, therefore some features of the website may not be accessible.</div>
-  </noscript>
-</body>
+        <div tal:condition="python:view.data.get('footer_text')" class="gallery-footer">
```

Comment:
here you are asking 3 times for `footer_text` it would be better to initialize a variable, even as this may be not costly in terms of performance.